docs/BUILD.md

```diff
@@ -0,0 +1,253 @@
+# Building the App for App Store Submission
+
+First, clone the repo (requires `git` - see [Dependencies](#dependencies)):
+
+```
+git clone https://github.com/WorldHealthOrganization/app.git
```

Comment:
```suggestion
git clone -b TAG_NAME https://github.com/WorldHealthOrganization/app.git
```

Reply:
Why use git? 
They can directly get the tag from here: 
`https://github.com/WorldHealthOrganization/app/archive/{{TAG}}.tar.gz`